Add `osu!mania LN Tournament 3` wiki article (#7987)
ppy/osu-wiki: wiki/Tournaments/o!mLN/3/en.md

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2022-09-19/2022-10-02 |
| Screening phase | 2022-10-02/2022-10-09 |
| Live drawings | 2022-10-09 |
| Qualifiers | 2022-10-15/2022-10-16 |
| Round of 64 | 2022-10-22/2022-10-23 |
| Round of 32 | 2022-10-29/2022-10-30 |
| Round of 16 | 2022-11-05/2022-11-06 |
| Quarterfinals | 2022-11-12/2022-11-13 |
| Semifinals | 2022-11-19/2022-11-20 |
| Finals | 2022-11-26/2022-11-27 |
| Grand Finals | 2022-12-03/2022-12-04 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge (TBC), 50% of final prize pool |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 30% of final prize pool |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 20% of final prize pool |

> *Note: The prize pool has yet to be finalised. If any player/spectator of the tournament is interested in donating to the prize pool, please contact any of the organiser for more information.*

Reviewer: ```suggestion
*Note: The prize pool has yet to be finalised. If any player/spectator of the tournament is interested in donating to the prize pool, please contact any of the organisers for more information.*
```

| Designer | ::{ flag=US }:: [Celsea](https://osu.ppy.sh/users/7464885), ::{ flag=ID }:: [nayuu](https://osu.ppy.sh/users/12561379) |
| Statistician | ::{ flag=VN }:: [MashedPotato](https://osu.ppy.sh/users/10494860) |
| Wiki writer | ::{ flag=ID }:: [fajar13k](https://osu.ppy.sh/users/7100002) |

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1633030)
- [o!mLN Discord server](https://discord.gg/SPQtt3s)
- [Livestream](https://www.twitch.tv/osumanialive)
- [Staff registration form](https://forms.gle/AgG6J3MEaREjsUFe6)
- **[Statistics sheet](https://docs.google.com/spreadsheets/d/11vi9CALEtaEnyvm-xwwGsudXB7HsaPMAoAXwF5hx0QY/edit?usp=sharing)**

## Participants

>  *Participants will be announced soon after the screening phase has ended.*

Reviewer: ```suggestion
 *Participants will be announced soon after the screening phase ends.*
```

## Mappools

>  *Every mappool will be announced 7 days (usually Sunday) before the next stage's match timeline.*

Reviewer: ```suggestion
*Every mappool will be announced 7 days (usually Sunday) before the next stage's match timeline.*
```

## Ruleset

### General rules

1. This tournament is a **1v1 double-elimination tournament**.
2. Players will first go through Qualifiers, then continuing to the double-elimination bracket.
3. Any player is allowed to sign up due to no rank limit.
4. Any members of staff **apart from organisers, mappool selectors, and referees** are allowed to play.
5. Players and staff members must have read this ruleset entirely.
6. Any rule changes or unexpected occurrences will be announced in the Discord server.

### Mappool format

The maps in this tournament will only consist of Long Note (LN) oriented maps, and cover 4 sub-categories of LNs which are used to categorise all mappools after this stage. See the graphics below for more information!

> ![mappool-categories](img/mappool-categories.jpg)
> 
> ![mappool-structure](img/mappool-structure.jpg)

Reviewer: ```suggestion
![Mappool categories](img/mappool-categories.jpg)

![Mappool structure](img/mappool-structure.jpg)
```

### Stage regulations

#### Qualifiers

> 1. after the screening phase has concluded, players eligible to play will participate in tournament play.

Reviewer: ```suggestion
1. After the screening phase has concluded, players eligible to play will participate in tournament play.
```

4. When both players are present in the lobby, the match will begin with each player playing their selected warmup (if any).

#### Mid-match procedure

1. After warmups, both players will run the `!roll` command in the lobby chat. The player with the higher roll will be able to **choose whether to protect, ban, and pick first or second**. The player with the lower roll receives the opposite of this.
2. Following the ban/pick procedure, both players will alternate in picks, and play the picked map simultaneously. The player who obtains a higher score will win a point, and this process repeats until one of the players wins the required number of points to claim the win for the match. The required number of points to win the match varies for each stage.
3. In the event where both players are 1 point away from winning the match, the tiebraeker will be played, and the player who wins the tiebreaker map will win the match.
4. Should there be a tie in map score between the 2 players, the result will be nullified and the map will be played once more to determine who gets the point.

> ![winning-conditions](img/winning-conditions.jpg)

Reviewer: ```suggestion
![Winning conditions](img/winning-conditions.jpg)
```
